platform/fabric/sdk/generic/providers.go

```diff
@@ -28,24 +28,6 @@ import (
 	"go.uber.org/dig"
 )
 
-type channelProviderResult struct {
-	dig.Out
-	generic.Provider `name:"generic"`
-}
-
-func NewChannelProvider(in struct {
-	dig.In
-	KVS                     *kvs.KVS
-	Publisher               events.Publisher
-	Hasher                  hash.Hasher
-	TracerProvider          trace.TracerProvider
-	Drivers                 []driver5.NamedDriver `group:"db-drivers"`
-	ChannelConfigProvider   driver.ChannelConfigProvider
-	ListenerManagerProvider driver.ListenerManagerProvider
-}) channelProviderResult {
-	return channelProviderResult{Provider: generic.NewProvider(in.KVS, in.Publisher, in.Hasher, in.TracerProvider, in.Drivers, in.ChannelConfigProvider, in.ListenerManagerProvider)}
-}
-
 type ChannelHandlerProviderResult struct {
```

Comment:
I remember that Alex suggested to keep all code that depends/uses on dig in a separate package to keep a structure where we see easily when dig is used.

@alexandrosfilios does this strategy still applies?